packages/blade/src/components/Alert/__tests__/__snapshots__/Alert.native.test.tsx.snap

```diff
@@ -29,12 +29,10 @@ exports[`<Alert /> should render 1`] = `
 >
   <View
     display="flex"
-    marginTop="spacing.0"
```

Comment:
Why are these getting removed tho? we just renamed to BaseBox right?

Reply:
nope. Rename PR is merged. This is snapshots updates for #1009 

I removed the margin and padding props top-level. They usually go and fail on react native when we pass `{ base: 'spacing.0', xs: '21px' }` something like this. They have a certain meaning when you put them on top-level and expect it to be string with `px` value only and can't take objects.

Anyway we only need them in styled-components to create style props so not needed on actual DOM